### ISSUE TITLE
Upload file programatically?

### ISSUE THREAD

**Issue by sorenwacker** [CCMS-UCSD/GNPSDocumentation#366]
Hi, I am the author of this application for metabolomics:

https://mint.resistancedb.org/
https://lewisresearchgroup.github.io/ms-mint-app/

I need some functionality to explore data files a bit, to verify they have data, look at tht TIC ect. 

I thought it might be possible to send the file to the GNPS dashboard. Is that maybe already implemented?
Or would that be a function you would be interested in supporting? 

I am thinking of a button in my app that sends the data file to the dashboard for viewing in a new browser tab.

**Comment by mwang87** [comment]
Hey Soren,

Yeah GNPS Dashboard would be great for this. One issue is where would the data be coming from. GNPS Dashboard would need to know a priori a type of data source (like a URL pattern) and then it'll be super easy. 

Best,

Ming

**Comment by sorenwacker** [comment]
My app can be installed and used locally. Even though there is this server application running. So there is no URL.

I thought, it may be possible to send a POST request with the file appended as data.

**Comment by mwang87** [comment]
Yes, there is the option available in the UI, but not specifically there in the API to make it nice and easy for you to use if someone can run on their own system. I think this is pretty easy to implement actually. 

I'm going to be pretty busy the next few weeks with Metabolomics 2022 coming up and moving to new faculty position at UCR, lets connect in early July if you're free and we can get it all worked out. 

Best,

Ming

**Comment by sorenwacker** [comment]
Oh, I wished I could be there. Have fun. Valencia right? Beautiful.

OK, I will be available.
